src/ansys/mapdl/core/_commands/prep7/areas.py

```diff
@@ -0,0 +1,1458 @@
+            area(s) so that they are consistent with each area's new normal direction.
+
+            * ``0`` - Make the normal direction of existing elements on the reoriented area(s) consistent with
+              each area's new normal direction (default).
+
+            * ``1`` - Do not change the normal direction of existing elements on the reoriented area(s).
+
+        Notes
+        -----
+
+        .. _ANORM_notes:
+
+        Reorients areas so that their normals are consistent with that of a specified area.
+
+        If any of the areas have inner loops, the :ref:`anorm` command will consider the inner loops when it
+        reorients the area normals.
+
+        You cannot use the :ref:`anorm` command to change the normal direction of any element that has a
+        body or surface load. We recommend that you apply all of your loads only after ensuring that the
+        element normal directions are acceptable.
+
+        Real constants (such as nonuniform shell thickness and tapered beam constants) may be invalidated by
+        an element reversal.
+
+        See `Revising YourModel
+        <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_mod/Hlp_G_MOD8_6.html>`_   of the
+        `Modeling and Meshing Guide
+        <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_mod/Hlp_G_MOD14.html>`_ for more
+        information.
+        """
+        command = f"ANORM,{anum},{noeflip}"
+        return self.run(command, **kwargs)
+
+    def asub(
+        self,
+        na1: str = "",
+        p1: str = "",
+        p2: str = "",
+        p3: str = "",
+        p4: str = "",
+        **kwargs,
+    ):
+        r"""Generates an area using the shape of an existing area.
+
+        Mechanical APDL Command: `ASUB <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_ASUB.html>`_
+
+        Parameters
+        ----------
+        na1 : str
+            Existing area number whose shape is to be used. If ``P1`` = P, graphical picking is enabled and
+            all remaining arguments are ignored (valid only in the GUI).
+
+        p1 : str
+            Keypoint defining starting corner of area.
+
+        p2 : str
+            Keypoint defining second corner of area.
+
+        p3 : str
+            Keypoint defining third corner of area.
+
+        p4 : str
+            Keypoint defining fourth corner of area (defaults to ``P3`` ).
+
+        Notes
+        -----
+
+        .. _ASUB_notes:
+
+        The new area will overlay the old area. Often used when the area to be subdivided consists of a
+        complex shape that was not generated in a single coordinate system. Keypoints and any corresponding
+        lines must lie on the existing area. Missing lines are generated to lie on the given area. The
+        active coordinate system is ignored.
+        """
+        command = f"ASUB,{na1},{p1},{p2},{p3},{p4}"
+        return self.run(command, **kwargs)
+
```

Comment:
git will normally track a changed file if it can map the old file to the new one as a modification. In this case, I think because the order of the commands is different, git doesn't realize that this is a move of `preproc/areas.py`, which makes this code harder to review.